### ISSUE TITLE
Question: should this work with Vite 3?

### ISSUE THREAD

**Issue by williamrjribeiro** [kyle-johnson/rollup-plugin-optimize-lodash-imports#307]
Hi there,

I have Vite 3 + React project that uses `lodash-es` and every import from the library follows the format `import isEmpty from 'lodash/isEmpty';`.

The thing is, many other used dependencies bring in `lodash` and I cringe when analyzing the production bundle with duplicated `lodash` functionality.

My hopes for this library were:

1. Just work with Vite 3
2. Reduce the production bundle size by optimizing *all* lodash imports.

Maybe my expectations are wrong? 🤷  Or maybe I'm overlooking some Vite 3 config detail? Well, any tips or clarifications are welcome.

This is my Vite config:
```js
import react from '@vitejs/plugin-react';
import { defineConfig } from 'vite';
import { visualizer } from "rollup-plugin-visualizer";
import { optimizeLodashImports } from "@optimize-lodash/rollup-plugin";

export default defineConfig({
  build: {
    rollupOptions: {
      plugins: [
        optimizeLodashImports({ useLodashEs: true }),
        visualizer({
          template: process.env.VIEW_BUNDLE,
          open: true,
          gzipSize: true,
        })
      ],
    }
  },

  plugins: [
    react(),
  ],
})
```

Cheers!

**Comment by kyle-johnson** [comment]
I’m guessing, but I suspect Vite tries to not process dependencies any further to speed up builds.

Any chance you have a clonable repo or branch showing this behavior?

**Comment by williamrjribeiro** [comment]
I'll create the repo and report back soon. 🙏

**Comment by kyle-johnson** [comment]
This probably has been fixed with the inclusion of #319

**Comment by kyle-johnson** [comment]
@williamrjribeiro this seems to work with the latest version (I didn't test older versions).

I have an example in #359 (I had success treating it as both a Vite `plugin` and adding it to `build.rollupOptions`). I just pushed a patch to include a section on usage with Vite.

Please drop a note if you're still experiencing an issue. :-)
